arm-datalake-store/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1321 @@
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/TrustedIdProviderProperties",
+          "description": "the properties of the trusted identity provider.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "description": "Data Lake Store firewall rule information"
+    },
+    "DataLakeStoreTrustedIdProviderListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/TrustedIdProvider"
+          },
+          "description": "the results of the list operation"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link (url) to the next page of results."
+        }
+      },
+      "description": "Data Lake Store trusted identity provider list information."
+    },
+    "DataLakeStoreFirewallRuleListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/FirewallRule"
+          },
+          "description": "the results of the list operation"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link (url) to the next page of results."
+        }
+      },
+      "description": "Data Lake Store firewall rule list information."
+    },
+    "EncryptionIdentity": {
+      "required": [
+        "type"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of encryption being used. Currently the only supported type is 'SystemAssigned'.",
+          "enum": [
+            "SystemAssigned"
+          ],
+          "x-ms-enum": {
+            "name": "EncryptionIdentityType",
+            "modelAsString": false
+          }
+        },
+        "principalId": {
+          "type": "string",
+          "format": "uuid",
+          "readOnly": true,
+          "description": "The principal identifier associated with the encryption."
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "readOnly": true,
+          "description": "The tenant identifier associated with the encryption."
+        }
+      },
+      "description": "The encryption identity properties."
+    },
+    "EncryptionConfig": {
+      "required": [
+        "type"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of encryption configuration being used. Currently the only supported types are 'UserManaged' and 'ServiceManaged'.",
+          "enum": [
+            "UserManaged",
+            "ServiceManaged"
+          ],
+          "x-ms-enum": {
+            "name": "EncryptionConfigType",
+            "modelAsString": false
+          }
+        },
+        "keyVaultMetaInfo": {
```

Comment:
No, because in the case where our service manages the encryption, they don't supply any key vault information because our service uses an internal keyvault for them.